myia/public_api.py

```diff
@@ -19,15 +19,14 @@
 from functools import reduce
 
 import numpy as np
-import torch
 
-from .. import operations
-from ..abstract import myia_static
-from ..hypermap import hyper_map
-from ..operations import primitives as P
-from ..utils import MyiaValueError, core
-from ..xtype import TupleT, f32, i64, u64
-from .pytorch_abstract_types import APT, APT_bool
+from . import operations
+from .abstract import myia_static
+from .frontends.pytorch_abstract_types import AA, APT, AS, APT_bool
```

Comment:
Consider using this instead of APT_bool:

```python
AA_bool = AbstractArray(
    AbstractScalar({TYPE: Bool, VALUE: ANYTHING}),
    {SHAPE: ANYTHING, TYPE: ANYTHING},
)
```